app/presenters/publishing_api/consultation_presenter.rb

```diff
@@ -0,0 +1,337 @@
+module PublishingApi
+  class ConsultationPresenter
+    extend Forwardable
+    include UpdateTypeHelper
+
+    SCHEMA_NAME = 'consultation'
+
+    attr_reader :update_type
+    def_delegator :consultation, :content_id
+
+    def initialize(consultation, update_type: nil)
+      self.consultation = consultation
+      self.update_type = update_type || default_update_type(consultation)
+    end
+
+    def content
+      BaseItemPresenter
+        .new(consultation)
+        .base_attributes
+        .merge(PayloadBuilder::PublicDocumentPath.for(consultation))
+        .merge(
+          description: consultation.summary,
+          details: details,
+          document_type: document_type,
+          public_updated_at: public_updated_at,
+          rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+          schema_name: SCHEMA_NAME,
+        )
+    end
+
+    def links
+      LinksPresenter
+        .new(consultation)
+        .extract(%i(organisations policy_areas topics))
+    end
+
+  private
+
+    attr_accessor :consultation
+    attr_writer :update_type
+    def_delegator :consultation, :display_type_key, :document_type
+
+    def base_details
+      {
+        body: body,
+        closing_date: consultation.closing_at,
+        emphasised_organisations: consultation.lead_organisations.map(&:content_id),
+        opening_date: consultation.opening_at,
+      }
+    end
+
+    def body
+      Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(consultation)
+    end
+
+    def details
+      base_details
+        .merge(ChangeHistory.for(consultation))
+        .merge(Documents.for(consultation))
+        .merge(ExternalURL.for(consultation))
+        .merge(FinalOutcome.for(consultation))
+        .merge(NationalApplicability.for(consultation))
+        .merge(PublicFeedback.for(consultation))
+        .merge(WaysToRespond.for(consultation))
+        .merge(PayloadBuilder::AccessLimitation.for(consultation))
```

Comment:
Is `AccessLimitation` in the correct place? Other formats expose it in `content` rather than `details`